### PR TITLE
hepmc: use actual tag

### PR DESCRIPTION
**Pull request by ktf**

### CODE REVIEW

**Comment by maireiphc** [comment]
Hello,
I don't get the problem : 2.06.10 exists in the official repository (https://gitlab.cern.ch/hepmc/HepMC/tree/2.06.10) ?
Should we try the tag format "HEPMC_02_06_10" (https://gitlab.cern.ch/hepmc/HepMC/tree/HEPMC_02_06_10) which seem to be more often used by the authors historically.

**Comment by maireiphc** [comment]
Hello,

  any news on this front, @ktf, @MichaelLettrich ?

Cheers,
    Antonin

File: hepmc.sh [hunk 1]
```diff
@@ -1,6 +1,6 @@
 package: HepMC
 version: "%(tag_basename)s"
-tag: v2.06.10
+tag: 2.06.10
 source: https://gitlab.cern.ch/hepmc/HepMC
```
Comment (maireiphc) [inline hepmc.sh]: The source has to be modified 
- from https://gitlab.cern.ch/hepmc/HepMC 
- to     https://gitlab.cern.ch/hepmc/HepMC.git

Note the ".git" missing at the end ; that is what prevents the HepMC sources to be downloaded apparently...
I have figured it out by looking at the alidist recipes provided for 'private' sources distributed via gitlab that we have in alisw, like epos.sh and dmpjet.sh.
This seems to cure the issue : it compiles under my alidock build at least.

File: hepmc.sh [hunk 2]
```diff
@@ -1,7 +1,7 @@
 package: HepMC
 version: "%(tag_basename)s"
-tag: v2.06.10
-source: https://gitlab.cern.ch/hepmc/HepMC
+tag: HEPMC_02_06_10
+source: https://gitlab.cern.ch/hepmc/HepMC.git
```
Comment (maireiphc) [inline hepmc.sh]: The source had to be modified

from 
    https://gitlab.cern.ch/hepmc/HepMC
to 
    https://gitlab.cern.ch/hepmc/HepMC.git

Note the ".git" missing at the end ; that is what prevents the HepMC sources to be downloaded apparently...
I have figured it out by looking at the alidist recipes provided for 'private' sources distributed via gitlab that we have in alisw, like epos.sh and dmpjet.sh.
This seems to cure the issue : it compiles under my alidock build at least.

(trying to recomment to figure out some features of GitBub Pull Request, would like to re-trigger the checks...)

Comment (ktf) [reply hepmc.sh]: It's enough that you do a commit or simply wait a bit. Anyways, it looks like it's working now, so I am merging.

Comment (maireiphc) [reply hepmc.sh]: Hello Giulio,
  thank you for this. Good piece of news.

Cheers,
  Antonin

**Pull request merged**